src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -38,6 +38,7 @@ interface FxaProfile {
   /** URL to an avatar image for the current user */
   avatar: string;
   avatarDefault: boolean;
+  subscriptions: Array<string>;
```

Comment:
When the user doesn't have any subscriptions, FxA doesn't add this property:

```suggestion
  subscriptions?: Array<string>;
```

For the other properties we'll either have to provide a fallback (`?? []`), or update those types as well to mark this one as optional.